fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/OrderByClause.java

```diff
@@ -27,7 +26,6 @@ public void add(String... expressions) {
 
     @Override
     public String toString() {
-        return items.stream().collect(Collectors.joining(", "));
+        return "ORDER BY " + items.stream().collect(Collectors.joining(", "));
```

Comment:
Could StringBuilder be used here instead of the "+" operator?

```
return new StringBuilder("ORDER BY ")
    .append(items.stream().collect(Collectors.joining(", ")))
    .toString();
```

Reply:
smooth... got it, changed it.